php-classes/Slate/Progress/AbstractSectionTermReportsRequestHandler.php

```diff
@@ -335,12 +335,31 @@ protected static function applyRequestFilters(array &$conditions = [], array &$r
 
         // optionally filter by course section
         if (!empty($_REQUEST['course_section'])) {
-            if (!$Section = Section::getByHandle($_REQUEST['course_section'])) {
-                return static::throwNotFoundError('course_section not found');
+            $courseSections = [];
+            if (!is_array($_REQUEST['course_section'])) {
+                $requestedCourseSections = [$_REQUEST['course_section']];
```

Comment:
@nbey let's use `explode(',', $_REQUEST['course_section'])` for turning the string into an array, as that will also let consumers pass comma-delimited arrays. This is a pattern we've followed in other places already